editor/export/editor_export_platform.cpp

```diff
@@ -621,7 +621,7 @@ HashSet<String> EditorExportPlatform::get_features(const Ref<EditorExportPreset>
 		}
 	}
 
-	return result;
+	return HashSet<String>(result);
```

Comment:
I don't think this change is correct, this should be copy-elided.


```suggestion
	return result;
```